applicationinsights-react-native/package.json

```diff
@@ -4,8 +4,8 @@
     "description": "Microsoft Application Insights React Native Plugin",
     "main": "dist-esm/index.js",
     "exports": {
-        ".": "dist-esm/index.js",
-        "./manual": "dist-esm/manualIndex.js"
+        ".": "./dist-esm/index.js",
+        "./dist-esm/manualIndex": "./dist-esm/manualIndex.js"
```

Comment:
Can we keep this as importing `./manual` as the "import" name as I would want to keep a separation from what the supported exported values are (ie `manual`) vs where the file is actually stored as part of the build `./dist-esm/manualIndex.js`